addons/udes_stock/models/stock_location.py

```diff
@@ -6,6 +6,7 @@ class StockLocation(models.Model):
     _name = "stock.location"
     # Add messages and abstract model to locations
     _inherit = ["stock.location", "mail.thread", "mixin.stock.model"]
+    MSM_STR_DOMAIN = ("name", "barcode")
```

Comment:
What is MSM?

Reply:
Dont know what sands for tbh, but is used to find the record from identifier.

Reply:
It stands for Mixin Stock Model, common code defined [here](https://github.com/unipartdigital/udes-open/blob/c585241ca2f751986e41941e174249a4a631c8d8/addons/udes_stock/models/mixin_stock_model.py#L10).

Reply:
mainstream media :laughing: